stable/locust/Chart.yaml

```diff
@@ -1,5 +1,7 @@
 name: locust
 description: A chart for locust distributed load testing
```

Comment:
I updated this to "A modern load testing framework" from the website, but up to you. As long as we remove the "A chart for" part, anything is fine.